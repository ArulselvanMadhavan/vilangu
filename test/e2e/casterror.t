int main() {
  int a[], b[][];
  Object c;
  a = new int[5];
  b = new int[5][];
  c = b;
  a = (int[])b;
}