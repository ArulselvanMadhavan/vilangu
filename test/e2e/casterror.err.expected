./test/e2e/casterror.t:7:6 - 7:14: cast not possible
